### ISSUE TITLE
[Feature Request] Inline preview & jump to definition for Helm Chart values

### ISSUE THREAD

**Issue by thewilli** [vscode-kubernetes-tools/vscode-kubernetes-tools#1084]
Jetbrains has great [support](https://blog.jetbrains.com/idea/2018/10/intellij-idea-2018-3-helm-support/) for Helm Chart development in their IDEs. Two features are related and really great - and I would really recommend to consider implementing them:

* There is no need to run an additional *preview* command, as the (default) values can just be seen inline in the template
*  It is possible to jump to a value's definition by clicking on a reference in a template

This is how it looks in a Jetbrains IDE:

![idea-k8sNavigation](https://user-images.githubusercontent.com/1206639/171653126-34f27be6-d629-4698-92d1-79b7b021b9d2.gif)

**Comment by mecampbellsoup** [comment]
YES, please do this, this is a big discrepancy relative to the Intellij plugin.

**Comment by magiceses** [comment]
+1

**Comment by mecampbellsoup** [comment]
Additionally auto-uncompressing the `.tgz` charts is something the IntelliJ version does which is very useful.

**Comment by svdimchenko** [comment]
this works great in intellij, and it would be super useful to add for vscode 💯

**Comment by hxnir** [comment]
Would love to see this feature!

**Comment by mecampbellsoup** [comment]
@thewilli any updates or expected timeline for this? Thank you kindly! 🙏
